brightwind/analyse/analyse.py

```diff
@@ -479,19 +479,16 @@ def custom_agg(x):
         graph, table12x24 = bw.twelve_by_24(df.PrcpTot, aggregation_method=custom_agg, return_data=True)
 
     """
+    var_name_label = var_series.name if var_name_label == "" else var_name_label
```

Comment:
This won't work if series is sent in the form df[['T2m']] (as a dataframe) and by convention we will set the default of var_name_label as None instead of ''. Implemented these changes in the commit after this.